server/src/api/drink/drinkRouter.js

```diff
@@ -0,0 +1,8 @@
+const router = require('express').Router();
+const db = require('../../db/store');
+const controller = require('./drinkController')(db);
+
+// get /drinks
+router.route('/').get(controller.getAll);
```

Comment:
With this folder structure, isn't ever single Router file going to look the same regardless of component? Just the route path and then the corresponding controller function?

Reply:
Yes. They will most likely end up looking extremely similar, if not the same. The difference will be which controller they use, or what the actual paths are, if they differ at all.

There's not really much cleaner of a solution that I can think of off the top my my head. There's not a cleanway to require files based on variables. The other option would be to put all of the routes in a single file which isn't any cleaner. Yes, there seems to be some duplicate code but all the routes have to be there anyways. This way, the components can stand alone as individuals. The loose coupling is the main benefit of this structure.